docs/Execution.md

```diff
@@ -1,92 +1,133 @@
+- Send a PubSub message to FCA's client follwoing the instructions in the [PubSub Handlers documentation](pubSubHandlers/PubSubHandlers.md).
+
+- **Sample Intent Message**:
+A sample intent message to trigger FCA remotely is provided below:
+
+```
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "APIName",
+    "params": {
+        "appId": "AppId",
+        "intent": {
+            "action": "search",
+            "data": {
+                "query": {
+                    "task": "runTest",
+                    "params": {
+                        "certification": true,
+                        "exceptionMethods": [
+                            {
+                                "method": "Api.method",
+                                "param": {
+                                    "type": "distributor"
+                                }
+                            },
+                            "..."
+                        ],
+                        "methodsToBeExcluded": [
+                            "Api.method",
+                            "Api.method",
+                            "..."
+                        ],
+                        "modulesToBeExcluded": []
+                    },
+                    "action": "CORE",
+                    "context": {
+                        "communicationMode": "SDK"
+                    },
+                    "metadata": {
+                        "target": "targetName",
+                        "targetVersion": "targetVersion",
+                        "...": "..."
+                    },
+                    "asynchronous": false,
+                    "appType": "firebolt"
+                }
+            },
+            "context": {
+                "source": "device"
+            }
+        }
+    }
+}
+
+```
+
+for more specific triggering follow instructions 
+[RunTest Handler documentation](intentReaderHandlers/RunTestHandler.md).
+
+
+# Test Types
+
+## Sanity
+
+Sanity testing ensures that all APIs function correctly at a basic level, serving as a quick check method.
+Execute the Sanity Test Suite from the UI, as described in the 'Supported Ways of Execution' section.
+This suite tests all APIs by invoking predefined calls under each API, offering a comprehensive overview of basic functionality.
+
+## Regression
+
+Regression testing involves in-depth testing to ensure new features or changes haven't adversely affected existing functionality.
+Create custom test cases with specific parameters tailored to the features or changes being tested.
+Send requests with custom parameters and verify responses against predefined data sets for targeted testing.
+to trigger those tests, build a framework communicating with your device, and send requests and verify responses as needed.
+
+
+# Modes of communication
+
+Different modes of communication offer flexibility in test approach, tailored to specific testing needs.
```

Comment:
Maybe we can provide a couple use cases to give context and highlight what we mean by `flexibility in test approach, tailored to specific testing needs.`

Reply:
@parag-pv do we have anything in ThorTV that can manifest the difference between communication via the SDK or directly with the device ?
only use cases we have is suits, not sure how we can give an example here.

Reply:
You could mention test scenarios where the API call should not go through the SDK and scenarios that should. When in transport mode, the SDK is bypassed which would be desirable in some use cases. Example would be a case where SDK would reject a requests based on the validations that it imposes.

Reply:
updated descriptions to this :
Different modes of communication offer flexibility in test approach, tailored to specific testing needs, for example invoking a device.version request would return SDK version if ran via SDK communication mode, and it'll be skipped if invoked in communication mode.

Reply:
So if invoked in Transport do we not get the `device.version`? Or does it get the actual version of the device?

Reply:
it get's skipped, we don't get the device version.

Reply:
@omarsabha91, rather than mentioning `skipped`, which might be confusing, could you rephrase it to convey that the device.version response in transport mode will not contain the `sdk` field in response but SDK mode will? 

The current doc seems to covey that transport mode will not give a response for device.version

Reply:
updated the description to:
Different modes of communication offer flexibility in test approach, tailored to specific testing needs, for example invoking a device.version request would return SDK version if ran via SDK communication mode, whereas in Transport mode the response will not contain `sdk` field.